versions/v1/units.json

```
@@ -6502,6 +6502,27 @@
     "source": "Custom based on qudt.org - https://qudt.org/vocab/unit/KiloPA-PER-M",
     "sourceReference": null
   },
+  {
+    "externalId": "pressure_gradient:psi-per-m",
+    "name": "PSI-PER-M",
+    "quantity": "Pressure Gradient",
+    "longName": "Pound Force per Square Inch per Meter",
+    "aliasNames": [
+      "psi per m",
+      "psi per meter",
+      "psi/m",
+      "PSI/M",
+      "psi / m",
+      "PSI / m"
+    ],
+    "symbol": "psi/m",
+    "conversion": {
+      "multiplier": 6894.76,
```

Here we could use the same conversion multipler in `pressure:psi`
```suggestion
      "multiplier": 6894.75789,
```